qtbooster/qtbooster.pro

```diff
@@ -1,18 +0,0 @@
-TEMPLATE = app
-
-TARGET = booster-qt5
-
-target.path = /usr/libexec/mapplauncherd/
-INSTALLS += target
-
-# Resolve all relocations at launch time.
-QMAKE_LFLAGS += -Wl,-z,now
-
-QT += network dbus xml sql 
-
-CONFIG += link_pkgconfig
-PKGCONFIG += glib-2.0 gobject-2.0 gmodule-2.0 gio-2.0 libffi sqlite3 libxml-2.0 applauncherd
```

Comment:
This is missing in the cmake variant.

Reply:
The `pkgconfig += applauncherd` is already there.
It looks like the others such as `QT+= network dbus xml sql ` and `PKGCONFIG += glib-2.0 gobject-2.0 gmodule-2.0 gio-2.0 libffi sqlite3 libxml-2.0` are not used in the code (at least not explicitly). Are they really needed?